app/src/main/java/com/nimbl3/ui/second/SecondActivity.kt

```diff
@@ -66,7 +63,7 @@ class SecondActivity : BaseActivity() {
 
     override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
-        onRequestPermissionsResult(requestCode, grantResults)
+        onRequestPermissionsResult(requestCode, permissions ,grantResults)
```

Comment:
small formatting issue `permissions ,` -> `permissions, `

Reply:
yup

Reply:
noted, thanks for spotting